source/UIAHandler/__init__.py

```diff
@@ -441,78 +444,85 @@ def terminate(self):
 			)
 		)
 		self.MTAThreadQueue.put_nowait(None)
-		#Wait for the MTA thread to die (while still message pumping)
-		if windll.user32.MsgWaitForMultipleObjects(1,byref(MTAThreadHandle),False,200,0)!=0:
+		# Wait for the MTA thread to die (while still message pumping)
+		if windll.user32.MsgWaitForMultipleObjects(1, byref(MTAThreadHandle), False, 200, 0) != 0:
 			log.debugWarning("Timeout or error while waiting for UIAHandler MTA thread")
 		windll.kernel32.CloseHandle(MTAThreadHandle)
 		del self.MTAThread
 
 	def MTAThreadFunc(self):
 		try:
-			oledll.ole32.CoInitializeEx(None,comtypes.COINIT_MULTITHREADED) 
-			isUIA8=False
-			try:
-				self.clientObject=CoCreateInstance(CUIAutomation8._reg_clsid_,interface=IUIAutomation,clsctx=CLSCTX_INPROC_SERVER)
-				isUIA8=True
-			except (COMError,WindowsError,NameError):
-				self.clientObject=CoCreateInstance(CUIAutomation._reg_clsid_,interface=IUIAutomation,clsctx=CLSCTX_INPROC_SERVER)
+			oledll.ole32.CoInitializeEx(None, comtypes.COINIT_MULTITHREADED)
+			self.clientObject = CoCreateInstance(
+				UIA.CUIAutomation8._reg_clsid_,
+				interface=UIA.IUIAutomation,

```

Comment:
```suggestion
				interface=UIA.CUIAutomation8._com_interfaces_[0],
```

Reply:
This will start with IUIA2 (Windows 8), and since minimum Windows version is 8.1, we can actually start with IUIA3 (index = 1). Wil add a comment to that effect.